Lib/test/test_os.py

```diff
@@ -227,6 +227,41 @@ def test_fdopen(self):
         self.fdopen_helper('r')
         self.fdopen_helper('r', 100)
 
+    @unittest.skipUnless(hasattr(os, 'O_DIRECT'), 'test needs os.O_DIRECT')
+    def test_odirect_readinto(self):
+        fname = support.TESTFN
+        create_file(fname, bytearray(mmap.PAGESIZE*2))
+        self.addCleanup(support.unlink, fname)
+
+        # Test one page readinto
+        fd = os.open(fname, os.O_DIRECT|os.O_RDONLY)
```

Comment:
PEP 8, same remark for following code:

```suggestion
        fd = os.open(fname, os.O_DIRECT | os.O_RDONLY)
```